src/pages/index.astro

```diff
@@ -1,55 +1,103 @@
 ---
-import EmailForm from "../components/Email-Form.astro";
 import Layout from "../layouts/Layout.astro";
-import Hero from "../components/Hero.astro";
-import Features from "../components/Features.astro";
-import Notification from "../components/Notification.astro";
-import Footer from "../components/Footer.astro";
-import "../styles/index.css";
 ---
 
 <Layout title="Purple Bubble">
-  <main>
-    <div>
-      <script
-        type="text/javascript"
-        src="https://unpkg.com/@lottiefiles/lottie-player@1.6.0/dist/lottie-player.js"
-      ></script>
+  <section><p>We are Purple Bubble</p></section>
+  <section>
+    <p>
+      Made up of teens from around the globe we hope to create a new messageing
+      app that will change the way you think about privacy and annonimity. We
+      want to revolutionize the way messaging apps work and make it so that
+      instead of you being the product, you are the user and the app works for
+      you.
```

Comment:
```suggestion
      Made up of teens from around the globe, we hope to create a new messaging
      app that will change the way you think about privacy and anonymity. We
      want to revolutionize the way messaging apps work and make it so that
      instead of being the product, you're using a product that works for you.
```